libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java

```diff
@@ -129,4 +129,13 @@ public void telemetryInitializerMiddlewareNotLogActivitiesDisabled() {
         List<String> eventNames = eventNameCaptor.getAllValues();
         Assert.assertEquals(0, eventNames.size());
 	}
+
+	@Test
+    public void telemetryInitializerMiddlewareWithUndefinedContext() {
+        // Arrange
+	    BotTelemetryClient mockTelemetryClient = Mockito.mock(BotTelemetryClient.class);
```

Comment:
This line has a weird indentation

Reply:
Yes, that's weird how we see this here, but it's correct